Library/Formula/swatchbooker.rb

```diff
@@ -17,10 +21,16 @@ def install
       s.gsub! "/usr/lib", "#{HOMEBREW_PREFIX}/lib"
     end
 
-    system "python", "setup.py", "install", "--prefix=#{prefix}"
-    bin.env_script_all_files(libexec+"bin", :PYTHONPATH => ENV["PYTHONPATH"])
-    chmod 0755, libexec/"bin/swatchbooker"
-  end
+    ENV["PYTHONPATH"] = libexec/"vendor/lib/python2.7/site-packages"
+
+    resource("Pillow").stage
+      system "python", *Language::Python.setup_install_args(libexec/"vendor")
```

Comment:
Wrap this line in a block and give it to the line above; it should look like this:
```rb
resource("Pillow").stage do
  system "python", *Language::Python.setup_install_args(libexec/"vendor")
end
```

`Resource#stage` takes either a directory or a block. In the former case it’ll unpack (read: uncompress) in it. In the latter it’ll create a temporary directory, `cd` and unpack in it, call the block, then `cd` back to where it was previously. Here we’re using a block so that unpacking the resource doesn’t mess with the buildpath.

This is what [`ansible.rb`](https://github.com/Homebrew/homebrew/blob/8e50348c825f9322d783105295d0c8c61be37896/Library/Formula/ansible.rb#L551-L553) does except that it uses `resources` to access all resources instead of retrieving each resource with its name using `resource("<name>")`.